File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/Auth/AuthDto.cs
using eRezervisi.Core.Domain.Authorization;

namespace eRezervisi.Common.Dtos.Auth
{
    public class AuthDto
    {
        public string Username { get; set; } = null!;
        public string Password { get; set; } = null!;
        public ScopeType Scope { get; set; }
    }
}

File: erezervisi_desktop/erezervisi_desktop/lib/models/requests/auth/auth_dto.dart
class AuthDto {
  String username;
  String password;
  final num scope = 1;

  AuthDto({
    required this.username,
    required this.password,
  });

  Map<String, dynamic> toJson() {
    return {'username': username, 'password': password, 'scope': scope};
  }
}
